spall.h

```diff
@@ -118,28 +125,32 @@ extern "C" {
 #endif
 
 // Profile context
-SpallProfile SpallInit    (const char *filename, double timestamp_unit);
-void         SpallQuit    (SpallProfile *ctx);
+SpallProfile spall_init (const char *filename, double timestamp_unit);
```

Comment:
Petition to rename `spall_init` to `spall_init_file` -- makes more clear that you can init to other things if you define the right callbacks

`spall_quit` doesn't need this change because it uses the `close` callback on the context ( 🚀 !!!)